homeassistant/components/rituals_perfume_genie/const.py

```diff
@@ -2,10 +2,18 @@
 
 from datetime import timedelta
 
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
 DOMAIN = "rituals_perfume_genie"
 
+# Alt (API V1)
 ACCOUNT_HASH = "account_hash"
 
+# Neu (API V2):
+# HA-Standards
+USERNAME = CONF_USERNAME
+PASSWORD = CONF_PASSWORD
```

Comment:
There's no need to redefine constants like this. Just just `homeassistant.const.CONF_xxx` directly